src/net_processing.h

```diff
@@ -146,6 +176,16 @@ class PeerManager final : public CValidationInterface, public NetEventsInterface
     CTxMemPool& m_mempool;
 
     int64_t m_stale_tip_check_time; //!< Next time to check for stale tip
+
+    /** Protects m_peer_map */
+    Mutex m_peer_mutex;
+    /**
+     * Map of all Peer objects, keyed by peer id. This map is protected
+     * by the global m_peer_mutex. Once a shared pointer reference is
```

Comment:
322e87b6039877ed3457baea93cc87d78f3f4d96
```suggestion
     * by the m_peer_mutex. Once a shared pointer reference is
```

Reply:
good catch. Fixed.